SPECS/cpprest/cpprest.spec

```diff
@@ -46,10 +47,11 @@ project aims to help C++ developers connect to and interact with services.
 Development files.
 
 %prep
-%setup -n cpprestsdk-%{version}
+%setup -q -n cpprestsdk-%{version}
```

Comment:
spec-lint checks required this change, and the others where existing lines were moved.